youtube2zim/scraper.py

```diff
@@ -792,6 +792,16 @@ def make_html_files(self, actual_videos_ids):
             - videos/<videoId>/video.jpg            template
         """
 
+        def remove_unused_videos(used_videos):
+            used_video_ids = []
+            for video in used_videos:
+                used_video_ids.append(video["contentDetails"]["videoId"])
+            videos_dir_contents = [x for x in self.videos_dir.iterdir() if x.is_dir()]
```

Comment:
why create a separate list and then loop on it? If you're not gonna reuse it, just loop directly:
``` py
for fname in self.videos_dir.iterdir():
    if not fname.is_dir():
        continue
    # do your stuff
```